src/components/Buttons/BackButton/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { TouchableOpacityProps } from 'react-native';
+import { ArrowIcon } from '../../Icons/ArrowIcon';
+import { BaseButton } from '../BaseButton';
+
+export const BackButton = (props: TouchableOpacityProps) => {
+  return (
+    <BaseButton {...props}>
+      <ArrowIcon color="black" />
```

Comment:
Is it okay to hardcode black color like that? Would it look nice on dark theme, when background is possibly dark color?